File: src/main/java/senior/design/group10/objects/response/ReturnAdmin.java
package senior.design.group10.objects.response;

public class ReturnAdmin {
    private String name;
    private String badgeId;

    public ReturnAdmin(){}

    public ReturnAdmin( String name, String badgeId)
    {
        this.name = name;
        this.badgeId = badgeId;
    }

    public String getName() {
        return name;
    }

    public String getBadgeId() {
        return badgeId;
    }
}

File: admin_application/USWRIC_Admin_Application/USWRIC_Admin_Application/ReturnAdmin.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace USWRIC_Admin_Application.objects
{
    class ReturnAdmin
    {
        public string Name { get; private set; }
        public string BadgeId { get; private set; }

        public ReturnAdmin(string name, string badgeId)
        {
            Name = name;
            BadgeId = badgeId;
        }
    }
}
